web/src/routes/(user)/photos/+page.svelte

```diff
@@ -55,19 +56,21 @@
       <AddToAlbum />
       <AddToAlbum shared />
     </AssetSelectContextMenu>
-    <DeleteAssets
-      on:escape={() => (handleEscapeKey = true)}
-      onAssetDelete={(assetId) => assetStore.removeAsset(assetId)}
-    />
-    <AssetSelectContextMenu icon={mdiDotsVertical} title="Menu">
-      <FavoriteAction menuItem removeFavorite={isAllFavorite} />
-      <DownloadAction menuItem />
-      <ArchiveAction menuItem onArchive={(ids) => assetStore.removeAssets(ids)} />
-      {#if $selectedAssets.size > 1}
-        <StackAction onStack={(ids) => assetStore.removeAssets(ids)} />
-      {/if}
-      <AssetJobActions />
-    </AssetSelectContextMenu>
+    {#if !hasPartnerAssetsInSelection}
```

Comment:
This basically disables all bulk actions in the main timeline when partner sharing is enabled, which isn't great.

I wonder how hard it would be to update the `AssetSelectControlBar` to have a `getAssets` and a `getOwnedAssets`, then the actions themselves could use the right one. So, `DownloadAction` uses `getAssets` and `DeleteAssets` uses `getOwnedAssets()`.

The set of assets could be filtered automatically in the `AssetSelectControlBar` if you passed in a `ownerId={user.id}` property of something.